src/database/migrations/20250519081754-add-tenant-code-org-role-req.js

```
@@ -0,0 +1,130 @@
+
+		// Add unique index (without partial condition)
+		try {
+			/* 			await queryInterface.sequelize.query(`
+				CREATE UNIQUE INDEX unique_value_org_id_tenant_code
+				ON "${tableName}" (tenant_code, organization_id, value)
+				WHERE deleted_at IS NULL;
+			`) */
+		} catch (error) {
+			console.error('Failed to add unique index:', error)
+			throw error
+		}
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)